core/test/models/workarea/bulk_action/product_edit_test.rb

```diff
@@ -52,7 +52,16 @@ def test_updating_pricing_skus
         product_2 = create_product(variants: [{ sku: 'SKU3', regular: 15.00 }])
 
         edit = ProductEdit.new(
-          pricing: { msrp: '19.99', prices: [{ regular: '14.99' }] }
```

Comment:
One big difference here is that I've changed the `prices:` param to be a Hash rather than an Array. I took a look back into the history of this code, and it seems like the Array of prices was added in the first WIP implementation. We started off just trying to pull off an `accepts_nested_attributes_for :prices` thing, but that was not being set correctly in releases, so Duffy made a change that explicitly ran `price.update!` on the given params for each price in each SKU. By not changing the type, this meant that he didn't need to make any breaking changes to the form, which was definitely the right choice at the time. Now that we need to change the form anyway, I felt like there was no need keeping around this old API. It's difficult to work with, and makes sense in the modern bulk edit workflow. Changing the type to a Hash makes the resulting `apply_pricing!` code a lot cleaner and easier to read.